modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/actions/Actions.tsx

```diff
@@ -21,6 +26,16 @@ import '../../../css/Actions.scss';
 import sortItems from '../../utils/sortItems';
 import {IOrderable} from '../../utils/types';
 
+const CREATION_ACTIONS_ORDER_OBJECT_FIELD_NAME = Liferay.FeatureFlags[
+	'LPD-15729'
+]
+	? 'creationActionsOrder'
+	: 'fdsCreationActionsOrder';
+
+const ITEM_ACTIONS_ORDER_OBJECT_FIELD_NAME = Liferay.FeatureFlags['LPD-15729']
+	? 'itemActionsOrder'
+	: 'fdsItemActionsOrder';
```

Comment:
removing `fds` prefix here makes total sense as this is a DSM abstraction which does not leak into FDS as it just receives a list of actions :+1: